src/coordination/coordinator_instance.cpp

```diff
@@ -387,6 +387,7 @@ auto CoordinatorInstance::TryFailover() -> void {
     if (raft_state_.IsLockOpened() && raft_state_.IsLeader()) {
```

Comment:
I would rename PR - the main thing is durable cluster config

Reply:
Makes sense